server/controllers/userController.js

```diff
@@ -75,18 +76,19 @@ const signIn = asyncHandler(async (req, res) => {
 // @route   GET /api/users/profile
 // @access  Private
 const getUserProfile = asyncHandler(async (req, res) => {
-  const user = await User.findById(req.user._id);
+  // const user = await User.findById(req.user._id);
```

Comment:
Why did you comment out this code? Can it be removed or should it be uncommented?

Reply:
I commented it out because it goes through the isAuthenticated middleware first which attaches the user to the request object already. But now I'm wondering where that should be done... I guess it doesn't matter too much but I think it'd be easier to do in the isAuthenticated middleware since you have access to the decoded.id